packages/shared/components/ReviewItem.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, Card, HStack, Image, Text } from '@chakra-ui/react';
+import React from 'react';
+
+import SettingIcon from '../assets/icon_menu.svg';
+
+type ReviewItemProps = {
+  children: React.ReactNode;
+  content: string;
+  images: string[];
+  onClick: VoidFunction;
+};
+
+export default function ReviewItem({
+  children,
+  content,
+  images,
+  onClick,
+}: ReviewItemProps) {
```

Comment:
```suggestion
import { PropsWithChildren } from 'react';

import SettingIcon from '../assets/icon_menu.svg';

type ReviewItemProps = {
  content: string;
  images: string[];
  onClick: VoidFunction;
};

export default function ReviewItem({
  children,
  content,
  images,
  onClick,
}: PropsWithChildren<ReviewItemProps>) {
```

그대로 진행되어도 정상적으로 잘 작동이 되고, 개인의 취향이라고 생각되어 제안드립니다!! 🙋🏻‍♂️
이런 경우 전 `PropsWithChildren`을 활용하는데 혹시 어떻게 생각하시나요?? 🤔

Reply:
PropsWithChildren은 children이 옵셔널로 되어있는데 제가 생각한 ReviewItem은 children이 필수라고 생각해서 저는 제 방식으로 진행할 것 같습니다.

참고: https://velog.io/@kkojae91/PropsWithChildren%EB%8A%94-%EC%95%88%EC%A0%84%ED%95%9C-%ED%83%80%EC%9E%85%EC%9D%BC%EA%B9%8C
위 글에서 만든 StrictPropsWithChildren을 만드는 건 괜찮아보이네요

팀원들과 논의하는 것 자체는 좋아요 :)

Reply:
제 생각에도 `ReviewItem` 컴포넌트는 children 을 꼭 넣어야하도록 설계되어 있어서 현재 방식대로 유지하는 것이 좋을 것 같습니다!